src/rp2/rp2_main.py

```diff
@@ -68,6 +68,12 @@ def _rp2_main_internal(country: AbstractCountry) -> None:  # pylint: disable=too
 
     _setup_paths(parser=parser, configuration_file=args.configuration_file, input_file=args.input_file, output_dir=args.output_dir)
 
+    try:
```

Comment:
```suggestion
    # On certain platforms the mpdecimal system library is missing (see details at: https://github.com/eprbell/rp2/issues/25).
    try:
```
Let's add a comment pointing to the discussion.